SOPT-iOS/Projects/Features/HomeFeature/Sources/HomeScene/Coordinator/HomeCoordinator.swift

```diff
@@ -16,47 +16,39 @@ import BaseFeatureDependency
 import HomeFeatureInterface
 import WebFeature
 
-public enum HomeCoordinatorDestination {
-    case signIn
-    case notification
-    case setting(userType: UserType)
-    case attendance
-    case soptlog
-    case calendar
-    case poke(isNewUser: Bool)
-    
-    case webLink(url: String)
-    case deepLink(url: String)
-}
-
-public protocol HomeCoordinatorOutput {
-    var requestCoordinating: ((HomeCoordinatorDestination) -> Void)? { get set }
+public protocol HomeCoordinatorDelegate: AnyObject {
+    func homeCoordinator(_ coordinator: HomeCoordinator, didRequest destination: HomeCoordinatorDestination)
```

Comment:
`didRequest`라는 별칭을 붙여주신 이유가 궁금합니다!(`destination`이 좀 더 의미가 와닿는 것 같아서요)

Reply:
애플 내장 프레임워크와 유사하게 매개변수 레이블로 구체적인 동작을 명시해주신 것 같아요!
저는 재현님처럼 매개변수 레이블이 있는 것을 더 선호합니다.

Reply:
didRequest말고도 다른 parameter label 추천하고 갑니다
```suggestion 

  func homeCoordinator(_ coordinator: HomeCoordinator, shouldNavigateTo destination: HomeCoordinatorDestination)
  func homeCoordinator(_ coordinator: HomeCoordinator, navigateTo destination: HomeCoordinatorDestination)
  func homeCoordinator(_ coordinator: HomeCoordinator, to destination: HomeCoordinatorDestination)

```

destination을 서술하기엔 navigateTo와 같이 "_어디로~_"의 의미가 담긴 label이 더 어울릴 것 같아요.
didRequest의 인자로는 Request가 와야 자연스러울 것 같기도 하구요!

Reply:
네엥! 제 생각에도 `didRequest`보다는 석우 님이 추천해주신 `to`가 더 직관적인 것 같아요. 반영하겠습니다!